frontend/src/util/validators.js

```diff
@@ -25,7 +24,7 @@ export function validatePHN(phn) {
   return validatePHNFormat(phn)
 }
 
-function validatePHNFormat(phn) {
+export function validatePHNFormat(phn) {
```

Comment:
I don't think this needs to be exported. It was meant as a "private" function and we aren't using in AddGroupMember anymore.